.env

```diff
@@ -34,6 +34,10 @@ MAIL_PORT="25"
 SMTP_USERNAME="ofn"
 SMTP_PASSWORD="f00d"
 
+OPENID_APP_ID = "4461746120466f6f6420436f6e736f727469756d"
+OPENID_APP_SECRET = "secret"
+OPENID_REDIRECT_URI = "http://localhost:3000/user/spree_user/auth/openid_connect/callback"
```

Comment:
We'll need to pair the final PR with one on ofn-install to include these variables for each instance